pkg/apis/postgres-operator.crunchydata.com/v1beta1/instrumentation_types.go

```diff
@@ -159,6 +164,12 @@ type InstrumentationCustomQueries struct {
 	// +default="5s"
 	// +optional
 	CollectionInterval *Duration `json:"collectionInterval,omitempty"`
+
+	// The database to target with added custom queries.
+	// Default behavior is to target `postgres`.
+	// ---
+	// +optional
+	Database string `json:"database,omitempty"`
```

Comment:
Do we want the user to be able to specify multiple databases for a given batch of custom queries? :thinking:

Reply:
Want to table this discussion and get feedback from the demo tomorrow?